pkg/mapper/entity_restapi/r2e/r2e.go

```diff
@@ -39,11 +39,7 @@ func MapAttachment(a interface{}) (entity.Attachment, error) {
 			return e, fmt.Errorf("all key/value pairs should be string/string. invalid attachment format %s", spew.Sdump(a))
 		}
 		for k, v := range m {
```

Comment:
no need to copy the key/value pairs here, you can just do `e=m`, or even just `return m, nil`

Reply:
fixed